src/mocks/data.ts

```diff
@@ -646,3 +647,18 @@ export const workPackageResponse: WorkPackageResponse = {
   token:
     'gumRa5XE1Rm3tOJr3ocfA1F63rRUI2g6eCI0KY2Mv3epb28cZeylvaYsxRmgQRDboE2yOhtE4qxPhZgYz/Y7zR+hssBzq7Hg',
 };
+
+export const storageAliasDecodes: baseStorageAliasDecodes = {
```

Comment:
```suggestion
export const storageAliasDecodes: BaseStorageAliasDecodes = {
```

type names should be uppercase